extensions/wikia/IndexingPipeline/ConnectionBase.class.php

```diff
@@ -45,7 +45,7 @@ public function publish( $routingKey, $body ) {
 					'delivery_mode' => self::DURABLE_MESSAGE,
 					'expiration' => self::MESSAGE_TTL,
 					'app_id' => 'mediawiki',
-					'correlation_id' => RequestId::instance()->getRequestId(),
+					'trace_id' => WikiaTracer::instance()->getTraceId(),
```

Comment:
I'd keep the old name here - `correlation_id` is a standard name for AMQPMessage property